site/src/Librecores/ProjectRepoBundle/Entity/ProjectClassifier.php

```diff
@@ -0,0 +1,190 @@
+<?php
+
+namespace Librecores\ProjectRepoBundle\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * ProjectClassifier
+ *
+ * @ORM\Table(name="project_classifier")
+ * @ORM\Entity(repositoryClass="Librecores\ProjectRepoBundle\Repository\ProjectClassifierRepository")
+ */
+class ProjectClassifier
+{
+    /**
+     * @var int
+     *
+     * @ORM\Column(name="id", type="integer")
+     * @ORM\Id
+     * @ORM\GeneratedValue(strategy="AUTO")
+     */
+    private $id;
+
+    /**
+     * @var int
+     *
+     * @ORM\Column(name="parent_id", type="integer")
+     */
+    private $parentId;
```

Comment:
what's that used for?

Reply:
Use auto-populated reference entities instead of directly accessing database foreign key references.